rules-reviewed/openliberty/liberty_LibertyList.windup.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="liberty_LibertyList">
```

Comment:
Could the `id` be more meaningful and self-explanatory? something like `liberty_java_unavailable_technologies` sounds easier to understand what the ruleset is about.

Consider the any change to the ruleset's ID has to be reflected also in the file name.